cg/meta/archive/ddn/models.py

```diff
@@ -72,40 +69,16 @@ def add_repositories(self, source_prefix: str, destination_prefix: str):
                 source_prefix=source_prefix, destination_prefix=destination_prefix
             )
 
-    def model_dump(self, **kwargs) -> dict:
-        """Creates a correctly structured dict to be used as the request payload."""
-        payload: dict = super().model_dump(exclude={"files_to_transfer"})
-        payload["pathInfo"] = [miria_file.model_dump() for miria_file in self.files_to_transfer]
-        return payload
-
-    def post_request(self, url: str, headers: dict) -> "TransferResponse":
-        """Sends a request to the given url with, the given headers, and its own content as payload.
-
-        Arguments:
-            url: URL to which the POST goes to.
-            headers: Headers which are set in the request
-        Raises:
-            HTTPError if the response status is not successful.
-            ValidationError if the response does not conform to the expected response structure.
-        Returns:
-            The job ID of the launched transfer task.
-        """
-
-        LOG.info(get_request_log(self.model_dump()))
-
-        response: Response = APIRequest.api_request_from_content(
-            api_method=APIMethods.POST,
-            url=url,
-            headers=headers,
-            json=self.model_dump(),
-            verify=False,
-        )
-        response.raise_for_status()
-        return TransferResponse.model_validate(response.json())
 
+class ArchivalResponse(BaseModel):
+    """Model representing the response fields of an archive request to the Dataflow
+    API."""
+
+    job_id: int = Field(alias="jobId")
 
-class TransferResponse(BaseModel):
-    """Model representing th response fields of an archive or retrieve reqeust to the Dataflow
+
+class RetrievalResponse(BaseModel):
+    """Model representing the response fields of a retrieval reqeust to the Dataflow
```

Comment:
Is it for better typehinting to have two models holding the same information, i.e. a job_id? 

Can't they both be ```ResponseId```?

Reply:
I felt that this was clearer for me to read. Now we see what archival request will return and what the retrieval request will return. Also, ArchivalResponse and RetrievalResponse classes contain only a subset of the returned values from the endpoint. So we could potentially in the future have a need for them to have different fields as well, should we have a need to parse more of them from the responses.